tests/neo.UnitTests/IO/Json/UT_JPath.cs

```diff
@@ -54,6 +55,13 @@ public class UT_JPath
             ["data"] = null,
         };
 
+        [TestMethod]
```

Comment:
Could you also add a boundary test where the size of an intermediate array is approximately equal to the maximum allowed limit?

The size in case is `width^depth` (`64^6` in this case). So tests for `2^10` (width 10, depth 2) and `32^2` (width 2, depth 32) should pass and result in an array of 1024 empty objects. Note, that the value itself should also be changed in this case (amount of `[` should be equal to depth).

This will help to ensure `neo-go` stays compatible.

Reply:
My bad, the second test shouldn't pass because of the depth limit.